addons/base_import/static/src/js/import_action.js

```diff
@@ -591,7 +592,10 @@ StateMachine.create({
         { name: 'import', from: ['preview_success', 'results'], to: 'importing' },
         { name: 'import_succeeded', from: 'importing', to: 'imported'},
         { name: 'import_failed', from: 'importing', to: 'results' }
-    ]
+    ],
+    error: function(eventName, from, to, args, errorCode, errorMessage, originalException) {
+        console.error('event occur from ' + eventName + ': ' + errorMessage);
+    },
```

Comment:
No. The entire point of using a state machine is to have clear and complete transitions, there is no good reason why state transition errors should happen, if they do there's a bug in the code (a case not handled or some such) and it certainly shouldn't be shoved under the rug and hidden away.